lab2.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import time
+
+random_numbers = np.random.randint(1, 10001, 100)
+lenth_of_array = len(random_numbers)
+
+
+def first_method(numbers_tuple):
+    clock_start = time.perf_counter()
+    sorted_by_function = sorted(numbers_tuple,
+                                key=lambda tup: tup[1], reverse=True)
+    clock_stop = time.perf_counter()
+    print(
+        f"Sorting the array by function took {clock_stop - clock_start:0.10f} seconds")
+    print(f"Sorted indexes: {indexes(sorted_by_function)}")
+
+
+def second_method(numbers_tuple):
+    lst = len(numbers_tuple)
+    clock_start = time.perf_counter()
+    for i in range(0, lst):
+        for j in range(0, lst-i-1):
+            if (numbers_tuple[j][1] < numbers_tuple[j + 1][1]):
+                temp = numbers_tuple[j]
+                numbers_tuple[j] = numbers_tuple[j + 1]
+                numbers_tuple[j + 1] = temp
+    clock_stop = time.perf_counter()
+    print(
+        f"Sorting the array by function took {clock_stop - clock_start:0.10f} seconds")
+    print(f"Sorted indexes: {indexes(numbers_tuple)}")
+
+
+def three_larger_numbers_indexes(list_of_numbers, threshold_value):
```

Comment:
z nazwy usunąłbym indexes ale to może kwestia ind. preferencji w nazewnictwie. funkcja zwracajaca trzy pierwsze wartosci moglaby sie nazywac three_larger_numbers, a zwracac wartosci i indexy jako pary czy tam same indexy